aptos-move/aptos-release-builder/src/validate.rs

```diff
@@ -398,14 +309,14 @@ impl NetworkConfig {
             "--max-gas",
             "2000000",
         ];
-        assert!(std::process::Command::new("cargo")
-            .current_dir(root_path.as_path())
-            .stdout(Stdio::inherit())
-            .stderr(Stdio::inherit())
-            .args(args)
-            .output()?
-            .status
-            .success());
+
+        let rev = self.framework_git_rev.clone();
+        if let Some(rev) = &rev {
```

Comment:
nit: You can combine these 2 lines into

if let Some(rev) = &self.framework_git_rev.clone() {

Reply:
Ha this is for hacking the rust compiler :D . If you simply do a clone the rev will not live long enough (cuz the vector is a vector of &str so it needs the string to have lifetime of this function, not just an if statement)